src/pytest/common.py

```diff
@@ -1,7 +1,17 @@
+
+import time
+
 def getConnectionByEnv(env):
     conn = None
     if env.env == 'oss-cluster':
         conn = env.envRunner.getClusterConnection()
     else:
         conn = env.getConnection()
-    return conn
\ No newline at end of file
+    return conn
+
+def waitForIndex(env, idx):
```

Comment:
I think it's better to allow a flag of sync indexing after rdb instead of start changing tests. Also on 6.0.7 we will have the loaded event and we will be able to index as the rdb loading